python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -480,8 +477,10 @@ class RandomForestClassifier(Base):
             input_to_dev_array(X, order='F')
 
         if self.dtype == np.float64:
-            warnings.warn("In order to run predict on the GPU convert"
-                          " the data to float32")
+            warnings.warn("Model is being trained with float64 data, which \
+                          means only CPU based predict can be used. \
+                          To use GPU based predict train use float32 training"
```

Comment:
Kind of funny that it uses backslash for the first line break and separate quotes for the second.

Reply:
I've tended to favor the backslash lately instead of what I used to do with separate quotes, so this is a combination of old and new me :P